README.md

```diff
@@ -12,72 +12,70 @@ This package contains a [Reshuffle](https://github.com/reshufflehq/reshuffle)
 connector for connecting to Twitter as a
```

Comment:
I suggest to follow same pattern as other Readme for the intro (title, link to npm, link to code, etc)
e.g. https://github.com/reshufflehq/reshuffle-jira-connector